pf/tfbridge/provider.go

```diff
@@ -192,7 +204,7 @@ func (p *provider) Close() error {
 
 // Pkg fetches this provider's package.
 func (p *provider) PkgWithContext(_ context.Context) tokens.Package {
-	return tokens.Package(p.packageSpec.Name)
+	return p.pkgName
```

Comment:
I think we can make this simpler:

Looking through the generate chain, we see that `ProviderInfo.Name` is ultimately used as the package name:

`GenerateSchemaWithOptions`:

https://github.com/pulumi/pulumi-terraform-bridge/blob/0f7ccfdef31d3fddd4eb46c9770d460bbfbe81e3/pkg/tfgen/generate.go#L693-L714

`genPulumiSchema`:

https://github.com/pulumi/pulumi-terraform-bridge/blob/0f7ccfdef31d3fddd4eb46c9770d460bbfbe81e3/pkg/tfgen/generate_schema.go#L246-L250

`genPackageSpec`:

https://github.com/pulumi/pulumi-terraform-bridge/blob/0f7ccfdef31d3fddd4eb46c9770d460bbfbe81e3/pkg/tfgen/generate_schema.go#L258-L260

I think we can ignore the schema artifact here and just use:

```suggestion
	return tokens.Package(p.info.Name)
```